tests/integration/files/conf/master

```diff
@@ -103,16 +103,24 @@ discovery: false
 # set uses to 0 so the token
 # has unlimited uses available.
 vault:
-  url: http://127.0.0.1:8200
```

Comment:
It would be ideal to keep testing for the old configuration as well to ensure we didn't break backwards compatibility.

Reply:
I'm not 100% certain when this file actually gets used. The default vault config in `tests/pytest/conftest.py` is still there:

https://github.com/saltstack/salt/blob/eb71862449b93f496c678d892e0c0ad827278136/tests/pytests/conftest.py#L177-L181

Also notice that the old config syntax is explicitly tested in the runner integration tests:

https://github.com/lkubb/salt/blob/32f977335765c530c66417a78504fdfde3dbdabd/tests/pytests/integration/runners/test_vault.py#L1152-L1247

Reply:
I have reset this file to the previous state now.